client/my-sites/email/email-forwarding/email-forwarding-add-new-compact.jsx

```diff
@@ -97,27 +90,25 @@ class EmailForwardingAddNewCompact extends Component {
 		return (
 			<div className="email-forwarding__form-content">
 				<FormFieldset>
-					<FormLabel>{ translate( 'Emails Sent To' ) }</FormLabel>
+					<FormLabel>{ translate( 'Emails sent to' ) }</FormLabel>
```

Comment:
:information_source: String reuse speeds up translation and improves consistency. The following string might make a good alternative and has already been translated **30** times:
`translate( 'Emails sent to', { context: 'Form label: Email sent to [address] (will arrive at...)'} )` *ES Score: 12*
See 1 additional suggestions in the [PR translation status page](https://translate.wordpress.com/localci/status/automattic/wp-calypso/update/email-comparison-page)